h2o-genmodel/src/main/java/water/genmodel/BuildVersion.java

```diff
@@ -0,0 +1,10 @@
+package water.genmodel;
+
+public class BuildVersion extends AbstractBuildVersion {
```

Comment:
is this fix temporary or will this file get re-generated automatically on new releases?

Reply:
thanks for asking - this file shouldn't be committed and I need need to adjust .gitignore as well

Reply:
fixed, thanks for pointing it out